src/rules/export.js

```diff
@@ -137,7 +141,7 @@ module.exports = {
           for (let [name, nodes] of named) {
             if (nodes.size <= 1) continue
 
-            if (isTypescriptFile && isTypescriptFunctionOverloads(nodes)) continue
+            if (isTypescriptFunctionOverloads([...nodes])) continue
```

Comment:
why the spread into an array? is `nodes` not an Array, just an iterable? `Set` maybe? if so I might `Array.from(nodes)` to make it a little clearer. but definitely nitpicking, this is a great improvement

Reply:
why not pass the Set in directly, and convert it inside the function?

Reply:
Ready for review